linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/rule.yml

```diff
@@ -5,7 +5,7 @@ prodtype: fedora,ol7,ol8,ol9,rhcos4,rhel7,rhel8,rhel9,rhv4,sle12,sle15,ubuntu200
 title: 'System Audit Logs Must Have Mode 0640 or Less Permissive'
 
 description: |-
-    {{% if 'ol' in product or "rhel" in product %}}
```

Comment:
I'm not sure what are the default permissions in RHCOS, won't this rule require a remediation?

Reply:
I hope not, because it's not possible to remediate file permissions on RHCOS or on kube without patching either OCP or RHCOS :-)

But you have a point that we didn't have tests for some rules. I added them and then we'll have proof right in CI.